jobboard/templates/jobboard/board.html

```diff
@@ -17,7 +40,13 @@ <h4>{{user_as_recruiter.company}}</h4>
     </div>
     <div class="col-10 text-center">
       <div class="row">
-        {% if user_is_student %}
+        {% if searched_name and not searched_area %}
+        <h3>Jobs that match "{{searched_name}}"</h3>
+        {% elif searched_area and not searched_name %}
+        <h3>Jobs that match "{{searched_area}}"</h3>
+        {% elif searched_area and searched_name %}
+        <h3>Jobs that match "{{searched_name}} and "{{searched_area}}"</h3>
+        {% elif user_is_student %}
         <h3>Here are some jobs that match your profile</h3>
         {% else %}
         <h3>Here are jobs created in the last 6 months</h3>
```

Comment:
Adding indentation to the if s could really help understand the code.. Happened to me too automatically. Not sure why.. Fixed it manually :\

Reply:
You're right, for me it's because I use the "prettier" extension in vscode, it reformats html code automatically when saving but it doesn't format Django code additions nicely. I will fix it.